Lib/distutils/tests/test_bdist_wininst.py

```diff
@@ -11,6 +12,8 @@ class BuildWinInstTestCase(support.TempdirManager,
                            support.LoggingSilencer,
                            unittest.TestCase):
 
+    @unittest.skipIf(sys.platform == 'win32' and platform.machine() == 'ARM64', 
+                    "No wininst-14.0-arm64.exe")
```

Comment:
I'm not sure what wininst-*.exe does. 
I'm not sure whether it's better to skip this test, or find the sources for wininst-*.exe, build it, and check the .exe in.

Reply:
No, let's just declare `bdist_wininst` unsupported on ARM. It's for generating self-extracting package installers.

Reply:
If we just declare it unsupported on ARM does that mean no change to this file?
It looks like the stub with _unsupported attribute only gets copied in by layout.  
Which seems to mean that the _unsupported attribute is not set when buildbot runs.
Or am I missing something?